code/modules/food&drinks/drinks/drinks/drinkingglass.dm

```diff
@@ -8,6 +8,7 @@
 	volume = 50
 	burn_state = 0 //Burnable
 	burntime = 5
+	flags = OPENCONTAINER
```

Comment:
Technically not needed since  /obj/item/weapon/reagent_containers/food/drinks already has it.